.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before filing a bug please check the following: -->
+<!-- * There's no existing/similar bug report. -->
+<!-- * This bug report is about a single actionable bug. -->
+<!-- * This bug is about the Pro Git book, version 2, English language. -->
+<!-- * This bug is about the book as found on the [website](https://www.git-scm.com/book/en/v2) or the pdf. -->
+<!-- * If you found a issue in the pdf/epub/mobi files, you've checked if the problem is also present in the Pro Git book on the [website](https://www.git-scm.com/book/en/v2). -->
+<!-- * This bug is **not** about a translation, if so please file a bug with the translation project. You can find a table of translation projects here: [progit2/TRANSLATING.md](https://github.com/progit/progit2/blob/master/TRANSLATING.md) -->
+<!-- * This bug is **not** about the git-scm.com site, if so please file a bug here: [git-scm.com/issues/new](https://github.com/git/git-scm.com/issues/new) -->
+<!-- * This bug is **not** about git the program itself, if so please file a bug here: [git-scm.com/community](https://git-scm.com/community) -->
+<!-- * This bug is **not** about Git for Windows, if so please file a bug here: [git-for-windows/git](https://github.com/git-for-windows/git). -->
+
+**Which version of the book is affected?**
+<!-- It's important for us to know if the problem is in the source or in the tooling for the pdf/epub/mobi files. -->
+<!-- Therefore, please write whether the problem is with the files, the online book, or both. -->
+
+**Describe the bug:**
+<!-- A clear and concise description of what the bug is. -->
+
+**Steps to reproduce:**
+<!-- Please write the steps needed to reproduce the bug here. -->
+<!-- 1. Go to '...' -->
+<!-- 2. Click on '....' -->
+<!-- 3. Scroll down to '....' -->
+<!-- 4. See error -->
+
+**Expected behavior:**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots:**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Additional context:**
+<!-- Add any other context about the problem here. -->
+<!-- You can also put references to similar bugs here. -->
+
+**Desktop:**
+<!-- If you've used a desktop/laptop to access the content, please fill in this form. -->
```

Comment:
@jnavila 
I'm not sure the comments for the desktop/smartphone/e-books details are actually helpful. 
I tried my bug report template, and it just gets in the way of filling in the form.
Maybe the plain-text parts are clear enough?

Reply:
@jnavila I think you might have missed my comment above?

1. Would it be better to remove comments like ```<!-- [e.g. iPhone 6 -->``` altogether? 
1. Or do you want those comments moved and squashed into one line right under their respective ```<!-- If you've used a [snip] -->``` lines?

Reply:
Hi,

For me, that's the same as the checkboxes: keeping some examples is good to help guiding the reporter, but it's useless to use html comments. So keep the comments but just let them juste as text.

Reply:
If we convert the comments to plain text, that plain text will clutter up the filled in/finished issue report. Are you sure that's what you want? It might be too much of a nuisance for anybody reading the finished report afterwards.

Reply:
We can keep the explanations in comments but remove the "e.g."

Reply:
The helpers can be kept in comments but the text to be edited should be uncommented so that we make sure that it will appear, even unedited.